### PR TITLE
add user mock data

### PR DESCRIPTION
**Pull request by jn1118**
Added user mock data to firestore database

### CODE REVIEW

**Review by bjtrieu — CHANGES_REQUESTED** [review]
As on discord: the social media links might be changed to handles, we might be adding hobbies, and we might be adding number bedroom and bathroom

But in the meantime, you can address the comments I made :)

**Comment by aidandenlinger** [comment]
Some questions up in the air:
- should social media hold handles or full links?
- are we collecting preferred number of bedrooms/bathrooms? (@lmy0125 this may impact login page)
- are we collecting hobbies as strings, or not at all? (@lmy0125 same)
- mild request for it to be `lifestyle.smoking` instead of `lifestyle.smoke`, but it's not a big deal either way

File: src/firebase/data/userData.js [hunk 1]
```diff
@@ -0,0 +1,1299 @@
...
+    lifestyle: {
+      smoke: false,
+      children: false,
+      pets: false,
+      parties: false,
+      alcohol: false,
+      couples: false,
+    },
+    maxPropertyPrice: 1200,
+    maxDistanceToSchool: 1.5,
```
[the remark this answers is not in the document]
Comment (jn1118) [reply src/firebase/data/userData.js:105]: Fixed

File: src/firebase/userDb.js [hunk 2]
```diff
@@ -1,64 +1,95 @@
...
+      const docRef = await addDoc(collection(db, "properties"), property[key]);
+      console.log("Document written with ID: ", docRef);
+    } catch (e) {
+      console.error("Error adding document: ", e);
+    }
+  }
+}
+
+export async function postMockUser() {
+  console.log("inside mock user");
```
Comment (bjtrieu) [inline src/firebase/userDb.js:82]:
```suggestion
```
just so that there's no debug statements in main

**Review by bjtrieu — CHANGES_REQUESTED** [review]
just a couple more small changes and then good to merge

File: src/firebase/userDb.js (hunk 2 above)
Comment (jn1118) [reply src/firebase/userDb.js]: Removed

File: src/firebase/data/userData.js [hunk 3]
```diff
@@ -0,0 +1,1367 @@
...
+    firstName: "Joe",
+    lastName: "White",
+    gender: "Male",
+    college: "Marshall",
+    major: "Computer Science",
+    schoolYear: "Junior",
+    languages: ["English", "Spanish"],
+    bedtime: "23:00",
+    numBedroom: "1",
+    numBathroom: "1",
```
[the remark this answers is not in the document]
Comment (jn1118) [reply src/firebase/data/userData.js]: Fixed.

**Review by bjtrieu — APPROVED** [review]
LGTM!

**Pull request merged**